config.go

```diff
@@ -252,6 +252,8 @@ type config struct {
 
 	RejectPush bool `long:"rejectpush" description:"If true, lnd will not accept channel opening requests with non-zero push amounts. This should prevent accidental pushes to merchant nodes."`
 
+	StaggerInitialReconnect bool `long:"stagger-initial-reconnect" description:"If true, will randomly stagger reconnections to persistent peers on startup between 0s and 30s. Nodes with a large number of channels should use this to reduce large memory bursts when restarting."`
```

Comment:
Any reason for including the flag and not just making this the default behavior?

Reply:
I think this should be default behavior, I have a commit locally that I will push up that makes it so. 

We probably don’t want this behavior on mobile, as it may take 30s to reconnect. Mobile devs can set this to false since they’ll probably have far fewer channels as well. We also want to disable this for integration tests, so the more recent version will also set this to false.

Reply:
latest version makes this the default behavior and disables it for itests